whisper/message.go

```diff
@@ -32,7 +33,12 @@ func (self *Message) sign(key *ecdsa.PrivateKey) (err error) {
 
 func (self *Message) Recover() *ecdsa.PublicKey {
 	defer func() { recover() }() // in case of invalid sig
-	return crypto.SigToPub(self.hash(), self.Signature)
+	pub, err := crypto.SigToPub(self.hash(), self.Signature)
+	if err != nil {
+		glog.V(0).Infof("Could not get pubkey from signature: ", err)
```

Comment:
Please use `logger.Error` rather than `0`

@obscuren note to self fix this for `vm` too.